src/pages/tasks/TaskAssigneeSelectorModal.js

```diff
@@ -263,8 +263,11 @@ export default compose(
         session: {
             key: ONYXKEYS.SESSION,
         },
+    }),
+    withOnyx({
```

Comment:
We should preferably use only one withOnyx or suppress the lint warning of using multiple withOnyx. The former is preferable.

Reply:
Okay, I see [that PR](https://github.com/Expensify/App/pull/28866) was [reverted](https://github.com/Expensify/App/pull/31231) so this should be fine.